_includes/home-hero.html

```diff
@@ -8,7 +8,7 @@ <h1 class="title">{{ site.tagline }}</h1>
   {% if site.is_hub %}
     <div class="cta">
       <a class="button" href="{{ "/projects/" | relative_url }}">
-        <i class="icon far fa-search"></i>
+        <i class="icon fas fa-search"></i>
```

Comment:
We probably want to switch the font here depending on whether Pro is used or not. @strogonoff ?

Reply:
I believe I chose the paid version in the first place because not all icons required by the original design were available in the free version.

Before merging this implementation, I will double-check whether all icons the theme is using as of now are available in the free version (probably not); if not then update theme design and/or disable the icon-requiring widgets based on the “Pro” setting, and update docs accordingly.

(We’ll probably eventually switch the whole theme to another fully free icon framework, or better completely custom icons, as @ronaldtse expressed preference for this as the long-term solution rather than FontAwesome, but this would take a bit longer to implement.)

cc @coder2000

Reply:
I think using an existing icon package like Font Awesome is better in the long run as it allows site creators to use it as well in their pages. Custom icons may not cover all uses cases someone may want or be stylistically compatible with other packages. IMHO.

Reply:
@coder2000 I agree as far as convenience goes. In general, I’m in favor of FA myself, though there are valid arguments against a third-party dependency.

About stylistic compatibility—this theme provides a specific “skin” with some customization capabilities, so if eventually we graduate to custom-crafted icons made to fit extremely well our theme’s design variations then that shouldn’t a problem. (Unless the user wants to completely override the design we provide, causing the icons to no longer look right, but that would be impossible to cover.) IMO, frankly, Open Project sites don’t look so fine and unique that using custom-crafted icons would make a difference just yet, so this is a hypothetical.

Reply:
I agree to stick with FA for now, it's a hassle to maintain a custom set.